src/app/header/header.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from "@angular/core";
+import { MatIconRegistry } from "@angular/material/icon";
+import { DomSanitizer } from "@angular/platform-browser";
+
+@Component({
+  selector: "app-header",
+  templateUrl: "./header.component.html",
+  styleUrls: ["./header.component.css"],
+})
+export class HeaderComponent implements OnInit {
+  constructor(
+    private matIconRegistry: MatIconRegistry,
+    private domSanitizer: DomSanitizer
```

Comment:
Why is `domSanitizer` required?

Reply:
The `domSanitizer` prevents Cross Site Scripting security bugs (XSS) by sanitising the values passed. XSS is an attack in which the attacker injects malicious scripts in the code of a trusted application or web page. This is avoided by Angular is it sanitises any value passed in the code, but in some cases where some JavaScript file that contains dynamic content must be included, we can bypass the security provided by angular and then use `domSanitzer` to clean the values.